src/components/ChatDown/ChatDown.js

```diff
@@ -1,17 +1,20 @@
+// @ts-check
 import React, { useContext } from 'react';
 import PropTypes from 'prop-types';
 
 import { LoadingChannels } from '../Loading';
 import { TranslationContext } from '../../context';
 
+// @ts-ignore
```

Comment:
We can add something like
```typescript
declare module "*.png" {
   const value: string;
   export = value;
}
```
to our type definitions to avoid having to ignore asset imports.

Reply:
ah yeah, nice. adding..